Lib/multiprocessing/pool.py

```diff
@@ -701,16 +729,16 @@ def _set(self, i, obj):
 
 class MapResult(ApplyResult):
 
-    def __init__(self, cache, chunksize, length, callback, error_callback):
-        ApplyResult.__init__(self, cache, callback,
+    def __init__(self, pool, chunksize, length, callback, error_callback, cache=None):
```

Comment:
Same here.

Reply:
Check my answer in the previous comment.